tools/tsp-client/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@azure-tools/tsp-client",
+  "version": "1.0.0",
+  "description": "A tool to generate Azure SDKs from TypeSpec",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run clean && npm run build:tsc",
+    "build:tsc": "tsc",
+    "clean": "rimraf ./dist ./types",
+    "example": "npx ts-node src/index.ts -e openapi -m https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/OpenAI.Inference/main.tsp ./temp",
+    "test": "mocha"
+  },
+  "author": "Jeff Fisher <jeffish@microsoft.com>",
+  "license": "MIT",
+  "type": "module",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "bin": "cmd/tsp-client.js",
+  "files": [
+    "dist",
+    "cmd/tsp-client.js"
+  ],
+  "devDependencies": {
+    "@types/chai": "^4.3.5",
+    "@types/mocha": "^10.0.1",
+    "@types/node": "^20.4.8",
+    "chai": "^4.3.7",
+    "mocha": "^10.2.0",
+    "prettier": "^3.0.1",
+    "rimraf": "^5.0.1",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "@azure/core-rest-pipeline": "^1.12.0",
+    "chalk": "^5.3.0",
+    "yaml": "^2.3.1"
+  },
+  "peerDependencies": {
+    "@typespec/compiler": "^0.47.1"
```

```suggestion
    "@typespec/compiler": ">=0.47.1 <1.0.0"
```

I recommend using this version range, which is equivalent to `^0.47.1` but also allows pre-release versions like `0.48.0-dev.18`.  This is useful for testing dev builds of TypeSpec packages.